src/client/src/pages/sign-in/sign-in.ts

```diff
@@ -4,6 +4,7 @@ import {
 	userSignInValidationSchema,
 } from "@/modules/user/user.js";
 
+import { initPasswordToggle } from "~/shared/helpers/password-toggle";
```

Comment:
```suggestion
import { initPasswordToggle } from "~/shared/index.js";
```

create a file `helpers.ts`, reexport that function from there and then reexport it inside of the `shared/index.ts` from `./libs/helpers/helpers.js`